charts/oes/values.yaml

```diff
@@ -1,20 +1,80 @@
 #####################################################
 # OpsMx Enterprise configuration
 #####################################################
-OESDBImage: opsmxdev/ubi8-autopilot-db:v1.2.1
-OESImage: opsmxdev/ubi8-autopilot:v0.202004212200
 k8sServiceType: LoadBalancer
-k8sSecret: oes-repo
-# Feature List
 installOpenLdap: true
+
+# Values to create imagePullSecret
+k8sSecret: oes-repo
+imageCredentials:
+  registry: index.docker.io/v1/
+  username: dockerHubId  # Docker hub username
+  password: dockerHubPwd # Docker hub password
+  email: abc@xyz.com   # email corresponding to docker hub ID
+
+#######################################################
+# Install the Seperate Redis if installSpinnaker is set to false
 installSpinnaker: true
-persistence:
-  enabled: true
-  OESDBSize: 8Gi
-# Password for the admin user
-openldap:
-  adminPassword: OpsMx@123
-  omitClusterIP: true
+installRedis: true
+#######################################################
+
+oes:
+  #labels: {}
+  image:
+    gate:
+      repository: opsmx11/oes-gate
+      tag: v0.202007021055
+      pullPolicy: IfNotPresent
+    sapor:
+      repository: opsmx11/sapor
+      tag: v0.202007011757
+      pullPolicy: IfNotPresent
+    ui:
+      repository: opsmx11/oes-ui
+      tag: v0.202007011835
+      pullPolicy: IfNotPresent
+  config:
+    db:
+      dbUrl: jdbc:postgresql://db-opsmx:5432/opsmx
+      username: postgres
+      password: networks123
+      #backendPort: 8090
+    server:
+      baseUrl: http://sapor-svc:8085
+      enabled: true
+    spinnaker:
+      oesGateURL: http://106.51.137.146:31174/  # Value of the OES-GATE URL
+      oesUIcors: ^https?://(?:localhost|oes-ui-svc|opsmx.com)(?::[1-9]\d*)?/? # Value of the OES UI URL Regex
+      spinGateURL: http://106.51.137.146:31174/gate # Value of the Spinnaker-gate URL
+      spinExternalGateURL: http://106.51.137.146:31174/gate # Value of the Spinnaker URL to access spinnaker from UI
+      spinuser: dummyuser # Value of the Spinnaker Username (Deck)
+      spinpasswd: dummypwd # Value of the Spinnaker Password (Deck)
+      adminLoginEnabled: false
+    ldap:
+      enabled: true
```

Comment:
say we do not install ldap but configure with existing authentication - how will that be handled?

Reply:
If LDAP Enabled is set to true, user need to provide their own ldap details in the values.yaml under ldap section.So that oue OES-Gate will take the user's LDAP Configuration for the authentication.